statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -759,35 +748,36 @@ def update(self, params, transformed=True, set_params=True):
 
         if not transformed:
             params = self.transform_params(params)
-        if set_params:
-            self.params = params
-
-        if self.updater is not None:
-            self.updater(self, params)
 
         return params
 
     @classmethod
     def from_formula(cls, formula, data, subset=None):
         """
-        Not implemented for State space models
+        Not implemented for state space models
         """
         raise NotImplementedError
 
 
-class MLEResults(FilterResults, tsbase.TimeSeriesModelResults):
+class MLEResults(tsbase.TimeSeriesModelResults):
     r"""
     Class to hold results from fitting a state space model.
 
     Parameters
     ----------
-    model : Model instance
+    model : MLEModel instance
         The fitted model instance
+    params : array
+        Fitted parameters
+    ssm_result : KalmanFilter instance
+        The underlying state space model and Kalman filter output
```

Comment:
how about calling it `filter_results` instead of `ssm_results` since it would match the class name. (3 letters more)

IIRC I use `results_aux` or something like this for auxiliary regression, but those are standard model results.
I think `filter` in front sounds fine.  
However, I don't really know what a FilterResults is. (it holds the filtered state, based on the name)